libdebug/architectures/call_utilities_provider.py

```diff
@@ -0,0 +1,23 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2023-2024 Francesco Panebianco. All rights reserved.
```

Comment:
Copyright date

Reply:
Resolved in commit [cd65140](https://github.com/libdebug/libdebug/commit/cd65140a09ac3456547822c981eed7dce3310dc2)